crates/aptos/src/move_tool/fmt.rs

```diff
@@ -0,0 +1,147 @@
+// Copyright (c) Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    common::types::{CliCommand, CliError, CliTypedResult},
+    update::get_movefmt_path,
+};
+use async_trait::async_trait;
+use clap::{Args, Parser};
+use std::{collections::BTreeMap, path::PathBuf, process::Command};
+
+/// Format the Move source code.
+#[derive(Debug, Parser)]
+pub struct Fmt {
+    #[clap(flatten)]
+    pub command: FmtCommand,
+}
+
+#[derive(clap::ValueEnum, Clone, Copy, Debug, Eq, PartialEq, PartialOrd)]
+pub enum EmitMode {
+    Overwrite,
+    NewFile,
+    StdOut,
+    Diff,
+}
+
+#[derive(Debug, Args)]
+pub struct FmtCommand {
+    /// How to generate and show the result after reformatting
+    #[clap(long, value_enum, default_value = "overwrite")]
+    emit_mode: EmitMode,
+
+    /// Path to the file to be formatted
+    #[clap(long)]
+    file_path: Option<PathBuf>,
+
+    /// Path to the directory to be formatted
+    /// if neither file_path or dir_path is set, all move files in the current folder will be formatted
+    #[clap(long)]
+    dir_path: Option<PathBuf>,
+
+    /// Path for the configuration file
+    /// Recursively searches the given path for the
+    /// movefmt.toml config file
```

Comment:
This says movefmt.toml while the next comment says .movefmt.tml

Reply:
done